README.md

```diff
@@ -18,6 +18,14 @@ persists them to a configured collector component.
 |------------------------------------|-----------------------------------------------------------------------------------------|
 | [collector-http](./collector-http) | Implements the [OTLP/HTTP protocol](https://opentelemetry.io/docs/specs/otlp/#otlphttp) |
 
+### Encoders
+
+The encoder encodes brave spans into OTLP proto format.
+
+| Encoder                                   | Description                                    |
+|-------------------------------------------|------------------------------------------------|
+| [`OtelEncoder`](./encoder-brave)     | zipkin-reporter-brave `AsyncZipkinSpanHandler` |
```

Comment:
Should this be changed to `OtlpProtoV1Encoder`?

Reply:
Fixed in https://github.com/openzipkin-contrib/zipkin-otel/pull/9/commits/b66e3a659186872a4573d618434aa250c643ce4a